assets/src/edit-story/components/helpCenter/constants.js

```diff
@@ -36,6 +36,7 @@ import { __ } from '@wordpress/i18n';
 export const TIPS = {
   addBackgroundMedia: {
     title: __('Add background media', 'web-stories'),
+    figure: 'images/editor/helpCenter/Add_BG_Module_1.webm',
```

Comment:
For the CDN images, correct path will be `images/help-center/add_bg_module_1.webm`.

All lowercase.

Same for the other paths.